zegami_sdk/collection.py

```diff
@@ -305,28 +390,38 @@ def get_image_urls(self, rows, source=0, generate_signed_urls=False,
             return signed_route_urls
 
     def download_annotation(self, annotation_id):
-        """
-        Converts an annotation_id into downloaded annotation data.
+        """Converts an annotation_id into downloaded annotation data.
```

Comment:
Multiline docstrings in our other python repos have the opening `"""` on its own line (as per before this change).
I don't have a strong opinion about this, but it does seem better to have a consistent style.

Reply:
All of collection.py and annotation.py fixed up to suit this